src/langcheck/metrics/ja/reference_based_text_quality.py

```diff
@@ -50,27 +57,32 @@ def semantic_similarity(
         reference_outputs: The reference output(s)
         prompts: The prompts used to generate the output(s). Prompts are
             optional metadata and not used to calculate the metric.
-        embedding_model_type: The type of embedding model to use ('local' or
-            'openai'), default 'local'
+        embedding_model_type: The type of embedding model to use ('local',
+            'openai', or 'azure_openai'), default 'local'
+        openai_client: OpenAI or AzureOpenAI client, default None. If this is
+            None but `model_type` is 'openai' or 'azure_openai', we will
+            attempt to create a default client.
         openai_args: Dict of additional args to pass in to the
-            `openai.Embedding.create` function, default None
+            `client.embeddings.create` function, default None
```

Comment:
Single backticks show up as italic in the documentation, so we should use double backticks

```suggestion
            None but ``model_type`` is 'openai' or 'azure_openai', we will
            attempt to create a default client.
        openai_args: Dict of additional args to pass in to the
            ``client.embeddings.create`` function, default None
```

Reply:
Ah got it, changed everything to be double backticks